include/jemalloc/internal/hash.h

```diff
@@ -13,305 +13,394 @@
 /* Internal implementation. */
 static inline uint32_t
 hash_rotl_32(uint32_t x, int8_t r) {
-	return ((x << r) | (x >> (32 - r)));
+  return ((x << r) | (x >> (32 - r)));
```

Comment:
This entire file was imported with minimal changes from upstream in order to facilitate later merges from upstream.

Reply:
We can use:

```c++
// clang-format off
... whole file goes here
// clang-format on
```

to easily disable clang-format for just this file without having to create a "white-list".